src/http/server/handler.cr

```diff
@@ -23,7 +23,7 @@ module HTTP::Handler
     if next_handler = @next
       next_handler.call(context)
     else
-      context.response.status_code = 404
+      context.response.status = HTTP::Status::NOT_FOUND
```

Comment:
Optional: just use `:not_found` instead of the full enum name. Same in other places where the status is assigned.

Reply:
Same in specs, for example `Response.new(:ok)`.

Reply:
This is still unresolved, just FYI.